project/Build.scala

```diff
@@ -746,18 +748,29 @@ object Build {
   // until sbt/sbt#2402 is fixed (https://github.com/sbt/sbt/issues/2402)
   lazy val cleanSbtBridge = TaskKey[Unit]("cleanSbtBridge", "delete dotty-sbt-bridge cache")
 
+  def cleanSbtBridgeImpl(): Unit = {
+    val home = System.getProperty("user.home")
+    val sbtOrg = "org.scala-sbt"
+    val bridgePattern = s"*dotty-sbt-bridge*$dottyVersion*"
+
+    IO.delete((file(home) / ".sbt" / "1.0" / "zinc" / sbtOrg * bridgePattern).get)
+    IO.delete((file(home) / ".sbt"  / "boot" * "scala-*" / sbtOrg / "sbt" * "*" * bridgePattern).get)
```

Comment:
How safe is deleting wildcard files under the home directory? Looks a bit scary (both the code and especially that it's allowed). Tho apparently we already did it?

Reply:
The paths used are as precise as I can reasonably make them to limit the use of wildcards. Suggestions on making this better are welcome but I think it's fine as is.